### PR TITLE
rbd: new size is equal to original size

### PR DESCRIPTION
**Pull request by zy751713126**
Signed-off-by: Zheng Yin zhengyin@cmss.chinamobile.com

### CODE REVIEW

**Comment by zy751713126** [comment]
thanks，dillaman. I will just the rbd CLI.

**Comment by zy751713126** [comment]
retest this please

File: src/tools/rbd/action/Resize.cc [hunk 1]
```diff
@@ -71,6 +71,11 @@ int execute(const po::variables_map &vm,
     return r;
   }
 
+  if (info.size == size) {
+    std::cerr << "rbd: resize error: new size is equal to original size " << std::endl;
```
[the remark this answers is not in the document]
Comment (zy751713126) [reply src/tools/rbd/action/Resize.cc]: thanks, dillaman. I have removed it.

**Comment by zy751713126** [comment]
retest this please

**Comment by zy751713126** [comment]
retest this please

**Comment by ctheune** [comment]
We just stumbled over this and haven't been able to find the motivation for this. Where did the idea of idempotent commands and UNIX philosophy leave the train?

**Pull request merged**